testsuite/gateway/gateway_api/gateway.py

```
@@ -101,7 +102,12 @@ def get_tls_cert(self):
             return None
 
         tls_cert_secret_name = self.cert_secret_name
-        tls_cert_secret = self.openshift.get_secret(tls_cert_secret_name)
+        tls_cert_secret = None
+        try:
+            tls_cert_secret = self.openshift.get_secret(tls_cert_secret_name)
+        except oc.OpenShiftPythonException as e:
+            if "Expected a single object, but selected 0" in e.msg:
+                pytest.skip("cert-manager operator is not properly configured")
```

pytest.error or straight up assert, this needs to fail loudly